packages/tempest/src/lists.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { render } from 'react-dom'
+
+//import styles from './Lists.css'
+export class List extends React.Component {
```

Comment:
Change to 
`export class List extends React.Component<ListProps> {`
to reflect the props change above. You can then get all the attributes off of props in the render() function. (Again, see Button.tsx.)